Framework/Core/Windows/Devices/I2c/I2cEnums.cs

```diff
@@ -0,0 +1,21 @@
+namespace Windows.Devices.I2c
+{
+    public enum I2cBusSpeed
+    {
+        StandardMode = 0,   // 100kHz

```

Comment:
Microsoft coding standards (including in UWP APIs, which we're mirroring).  The first value should be explicitly assigned, with all following values implicit when possible.